src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -484,6 +484,16 @@ public sealed partial class LCIDConversionAttribute : System.Attribute
         public LCIDConversionAttribute(int lcid) { }
         public int Value { get { throw null; } }
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Method, AllowMultiple = false, Inherited=false)]
+    public sealed partial class LibraryImportAttribute : System.Attribute
+    {
+        public LibraryImportAttribute(string libraryName) { }
+        public string LibraryName { get { throw null; } }
+        public string? EntryPoint { get { throw null; } set { } }
+        public bool SetLastError { get { throw null; } set { }}
+        public StringMarshalling StringMarshalling { get { throw null; } set { } }
+        public Type? StringMarshallingCustomType { get { throw null; } set { } }
```

Comment:
These type names should be fully-qualified to match what the ref generator spits out.

Reply:
```suggestion
        public System.Runtime.InteropServices.StringMarshalling StringMarshalling { get { throw null; } set { } }
        public System.Type? StringMarshallingCustomType { get { throw null; } set { } }
```